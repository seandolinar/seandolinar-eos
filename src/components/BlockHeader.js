import React from 'react';


const BlockHeader = ({data, count}) => <>
    <div>{data.id}</div>
    <div>{data.timestamp}</div>
    <div>Total Transactions: {count.transactions}</div>
    <div>Actions: {count.actions}</div>
</>


export default BlockHeader