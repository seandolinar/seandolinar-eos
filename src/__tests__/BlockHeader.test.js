import React from 'react';
import renderer from 'react-test-renderer';

import BlockHeader from '../components/BlockHeader';

const mockProps = {
    data: {id: 'a01', timestamp: '2019-09-13T13:01:04.500'},
    count: {transactions: 9999, actions: 555}
}

it('matches previous snapshot', () => {
    expect(renderer.create(<BlockHeader {...mockProps} />).toJSON()).toMatchSnapshot();
});
