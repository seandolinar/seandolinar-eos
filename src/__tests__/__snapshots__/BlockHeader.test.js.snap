// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches previous snapshot 1`] = `
Array [
  <div>
    a01
  </div>,
  <div>
    2019-09-13T13:01:04.500
  </div>,
  <div>
    Total Transactions: 
    9999
  </div>,
  <div>
    Actions: 
    555
  </div>,
]
`;
